mario/tools/parsersclass.py

```diff
@@ -146,9 +167,25 @@ def parse_from_excel(
     mario.Database
     """
 
+    # check the inputs to be correct
+    errmsg = []
+    if not isinstance(path,str):
+        errmsg.append("Path should be a string")
+    if table not in _ACCEPTABLES['table']:
+        errmsg.append(f"Table should be in {_ACCEPTABLES['table']}")
```

Comment:
better to check it in a non-case sensitive model like:

if table.lower() not in ....

of course, only if it is not a hassle to do it

Reply:
let's proceed now with this, we can improve it